health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/ViewMap.js

```
@@ -0,0 +1,281 @@
+
+// Utility function to introduce a delay
+export const delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+export const removeAllLayers = (map) => {
+  if (!map || !map._layers) {
+    console.error("Map or layers are undefined");
+    return;
+  }
+  
+  Object.values(map._layers).forEach((layer) => {
+    if (layer && layer.remove) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)